subprojects/core/src/main/java/org/gradle/api/Project.java

```diff
@@ -1452,6 +1452,41 @@
     CopySpec copySpec();
 
     /**
+     * Synchronizes the contents of a destination directory with some source directories and files.
+     *
+     * <p>
+     * This task is like the {@link #copy(Action)} task, except the destination directory will only contain the files copied.
+     * All files that exist in the destination directory will be deleted before copying files, unless a preserve option is specified.
+     *
+     * <p>
+     * Examples:
+     * <pre autoTested=''>
+     * // Sync can be used like a Copy task
```

Comment:
Like `copy(Closure)`, we should have an example that uses `project.sync` instead of the `Sync` task.